{
	"cve": {
		"data_type": "CVE",
		"data_format": "MITRE",
		"data_version": "4.0",
		"CVE_data_meta": {
			"ID": "CVE-2022-25252",
			"ASSIGNER": "ics-cert@hq.dhs.gov"
		},
		"problemtype": {
			"problemtype_data": [
				{
					"description": []
				}
			]
		},
		"references": {
			"reference_data": [
				{
					"url": "https://www.ptc.com/en/support/article/CS363561",
					"name": "https://www.ptc.com/en/support/article/CS363561",
					"refsource": "MISC",
					"tags": []
				},
				{
					"url": "https://www.cisa.gov/uscert/ics/advisories/icsa-22-067-01",
					"name": "https://www.cisa.gov/uscert/ics/advisories/icsa-22-067-01",
					"refsource": "MISC",
					"tags": []
				}
			]
		},
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "When connecting to a certain port Axeda agent (All versions) and Axeda Desktop Server for Windows (All versions) when receiving certain input throws an exception. Services using said function do not handle the exception. Successful exploitation of this vulnerability could allow a remote unauthenticated attacker to crash the affected product."
				}
			]
		}
	},
	"configurations": {
		"CVE_data_version": "4.0",
		"nodes": []
	},
	"impact": {},
	"publishedDate": "2022-03-16T15:15Z",
	"lastModifiedDate": "2022-03-16T15:15Z"
}